### PR TITLE
Board: Checklists model

### PR DESCRIPTION
**Pull request by annano**

### CODE REVIEW

File: plugins/board/src/index.ts [hunk 1]
```diff
@@ -56,6 +56,23 @@ export interface CardDate extends Obj {
...
+  assignee?: Ref<Employee>
+  dueDate?: Timestamp
+  isChecked: boolean
+  name: Markup
+}
+
+/**
+ * @public
+ */
+export interface CardChecklist extends Obj {
```
Comment (BykhovDenis) [inline plugins/board/src/index.ts:72]: Why you extend Obj?

File: plugins/board/src/index.ts [hunk 2]
```diff
@@ -56,6 +56,23 @@ export interface CardDate extends Obj {
   startDate?: Timestamp
 }
 
+/**
+ * @public
+ */
+export interface CardChecklistItem extends Obj {
```
Comment (BykhovDenis) [inline plugins/board/src/index.ts:62]: Why you extend Obj?

File: models/board/src/index.ts [hunk 3]
```diff
@@ -70,6 +70,12 @@ export class TCardLabel extends TAttachedDoc implements CardLabel {
   isHidden?: boolean
 }
 
+@Model(board.class.CardChecklist, core.class.Obj, DOMAIN_MODEL)
+@UX(board.string.Checklist)
+export class TCardChecklist extends TObj implements CardChecklist {
```
Comment (BykhovDenis) [inline models/board/src/index.ts:75]: Why you need it in model?

**Review by BykhovDenis — CHANGES_REQUESTED** [review]

File: models/board/src/index.ts (hunk 3 above)
Comment (annano) [reply models/board/src/index.ts:75]: We would need to add presenters for this in the future to display it in activity

File: models/board/src/index.ts [hunk 4]
```diff
@@ -105,6 +111,9 @@ export class TCard extends TTask implements Card {
 
   @Prop(Collection(contact.class.Employee), board.string.Members)
   members?: Ref<Employee>[]
+
+  @Prop(Collection(board.class.CardChecklist), board.string.Checklist)
```
Comment (BykhovDenis) [inline models/board/src/index.ts:115]: It should be 
```
@Prop(ArrOf(TypeCardChecklist()), board.string.Checklist)


@Model(contact.class.TypeCardChecklist, core.class.Type)
export class TTypeCardChecklist extends TType {}

export function TypeCardChecklist (): Type<CardCheckList> {
  return { _class: board.class.TypeCardChecklist, label: board.string.Checklist }
}
 ```

**Pull request closed without merging**